torch/csrc/distributed/c10d/logger.cpp

```diff
@@ -8,7 +8,7 @@ namespace c10d {
 
 // When training runs at these iterations, log the runtime
 // stats.
-const int LoggingIterations[] = {10, 20, 100, 1000};
+const int LoggingIterations[] = {10, 20, 100, 1000, 5000, 10000, 20000};
```

Comment:
This seems like something we should be able to control with a config knob rather than needing to change code like this. Also, why do we have specific iterations like this instead of just logging based on a sampleRate?

Reply:
@pritamdamania87 Agreed we should be able to config this via a knob. I'll send a quick fix for @zhaojuanmao's comment for now as well

Reply:
Regarding specific iterations vs a sample rate, we currently have a sampleRate at which the logger does anything at all (i.e. it doesn't do any logging/time recording unless it is sampling at that iteration). In addition to that, these LoggingIterations are specific iterations at which the data will be dumped to its configured destination - https://github.com/pytorch/pytorch/blob/master/torch/csrc/distributed/c10d/logger.cpp#L360